functions/firebase/index.js

```diff
@@ -17,12 +17,13 @@
  * Triggered by a change to a Firebase RTDB reference.
  *
  * @param {!Object} event The Cloud Functions event.
+ * @param {!Object} context The Cloud Functions event context.
  */
-exports.helloRTDB = event => {
-  const triggerResource = event.resource;
+exports.helloRTDB = (event, context) => {
```

Comment:
nit: Link to find this info like https://cloud.google.com/functions/docs/writing/background#function_parameters

Reply:
Called out that these are `Background Functions` - that should be precise enough for users to find that page.